src/EntryPoints/MediaWikiHooks.php

```diff
@@ -94,4 +97,34 @@ private static function newEntityRdfBuilderFactoryFunction( callable $factoryFun
 		);
 	}
 
+	public static function onContentHandlerDefaultModelFor( Title $title, ?string &$model ): void {
+		if ( WikibaseRdfExtension::getInstance()->isConfigTitle( $title ) ) {
+			$model = CONTENT_MODEL_TEXT;
+		}
+	}
+
+	public static function onEditFilter( EditPage $editPage, ?string $text, ?string $section, string &$error ): void {
+		if ( is_string( $text ) && WikibaseRdfExtension::getInstance()->isConfigTitle( $editPage->getTitle() ) ) {
+			$validator = new PredicatesTextValidator();
+			if ( !$validator->validate( $text ) ) {
+				$invalidPredicates = implode(
```

Comment:
The message building here should probably move out. I don't think there is a nice MW way to pass in the actual array. I think you have to pass the final string to the message.